init/uninstall.pl

```diff
@@ -59,6 +59,10 @@ sub module_uninstall
 	# Delete win32 service
 	&delete_win32_service($product);
 	}
+elsif ($init_mode eq "systemd") {
```

Comment:
Similarly, what if the user had an init script before?

Reply:
Similarly, should we uninstall for both?

Reply:
Similarly, appears to be not a problem.

Reply:
The best option would be to to search for all kinds of init scripts, and remove whichever one the user has.

Reply:
Jamie, main old `/etc/init.d/webmin` script will be removed (cleared) automatically upon package upgrade (i.e. 1.990->1.991).

Reply:
Ok, cool. Is there any possible conflict if this file is removed upon upgrade, but then re-created by the postinstall script?

Reply:
> Is there any possible conflict if this file is removed upon upgrade, but then re-created by the postinstall script?

Not that I'm aware of! I don't see a way for `init.d/webmin` to be recreated with existing code found (in PR). Do you?